include/pybind11/attr.h

```diff
@@ -12,7 +12,7 @@
 
 #include "cast.h"
 
-NAMESPACE_BEGIN(pybind11)
+NAMESPACE_BEGIN(PYBIND11_NAMESPACE)
```

Comment:
Wouldn't it be simpler to add the visibility attribute to the `NAMESPACE_BEGIN` macro instead of introducing a new one?